#!/usr/bin/env python
PACKAGE = "wheeltec_yolo_action"
 
from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()

gen.add("HSV_H_min", int_t, 0, "HSV_H_min", 0, 0, 180)
gen.add("HSV_S_min", int_t, 0, "HSV_S_min", 0, 0, 255)
gen.add("HSV_V_min", int_t, 0, "HSV_V_min", 0,	0, 255)
gen.add("HSV_H_max", int_t, 0, "HSV_H_max", 180, 0, 180)
gen.add("HSV_S_max", int_t, 0, "HSV_S_max", 255, 0, 255)
gen.add("HSV_V_max", int_t, 0, "HSV_V_max", 255, 0, 255)

exit(gen.generate(PACKAGE, "wheeltec_yolo_action", "hsv"))  
